src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Ball {
+    private int number;
+
+    private Color color;
```

Comment:
You can simplify your logic by using constructor with arguments 
Ball(String color, int number)